### DISCUSSION TITLE
Retrieving data for scoring

### DISCUSSION THREAD

**Discussion by vebrt99** [GoogleChrome/lighthouse#15681] [Q&A]
I was quite deeply inspecting how metrics scores are calculating and I found out, that used data for scoring values (median and p10) are few years old. So there are my questions:

1. how often are those scoring values updating?
2. can i change those values myself for my audit?
3. what is the source of these values? Some of them are from HTTPArchive i guess, but i could not find them all there.

**Comment by adamraine** [comment]
> how often are those scoring values updating?

We don't have a set timeline for updating them.

> can i change those values myself for my audit?

Yes using a [custom config](https://github.com/GoogleChrome/lighthouse/blob/main/docs/configuration.md) like this:

```js
export default {
  extends: 'lighthouse:default',
  audits: [
    {path: 'metrics/largest-contentful-paint', options: {
      mobile: {
        scoring: {median: 500, p10: 100},
      },
      desktop: {
        scoring: {median: 500, p10: 100},
      },
    }},
  ],
};
```

> what is the source of these values? Some of them are from HTTPArchive i guess, but i could not find them all there.

Indeed HTTPArchive data is the primary source for these values. Each set of scoring values should have a comment explaining where they come from in the source code.

Example: [CLS scoring values](https://github.com/GoogleChrome/lighthouse/blob/main/core/audits/metrics/cumulative-layout-shift.js#L42-L44) were also influenced by manual evaluation.

**Reply by vebrt99** [reply]
Hello again,
in first place i would like to thank you for your answer.

I am interested in all of these metrics: FCP, LCP, TBT, CLS, SI, INP, TTI, FID, TTFB and I do have still few questions.
1. I was looking into the source code and found links to HTTPArchive, where source data for scoring should be from. But I was not able to find those values in there. From the path in url i found the file/database table (for example this url from first-contentful-paint.js https://bigquery.cloud.google.com/table/httparchive:lighthouse.2021_05_01_mobile) with only two attributes - URL and Report. So what am I doing wrong?
2. Are data for all the mentioned metrics available in HTTPArchive?
3. If the scoring in a metric source file is not divided for mobile and desktop, does it mean that there is no difference between them and those values are used for both of them?

Thank you in advance for your reply.
